app/components/Article/article.css

```diff
@@ -77,6 +76,15 @@ article .link-popup::after {
   right: 100%;
 }
 
+article .link-popup .glossary-popup > .contents {
+  padding: var(--spacing-24) var(--spacing-40) var(--spacing-24);
```

Comment:
2 or 4 values would make more sense here IMHO, but if the bottom is same as top, repeating it looks suspicious